cluster/src/main/java/org/apache/iotdb/cluster/server/member/DataGroupMember.java

```diff
@@ -204,11 +202,11 @@ public DataGroupMember(PartitionGroup nodes) {
     allNodes = nodes;
     setQueryManager(new ClusterQueryManager());
     slotManager = new SlotManager(ClusterConstant.SLOT_NUM, getMemberDir(), getName());
+    singleReplicaApplier = new DataLogApplier(metaGroupMember, this);
```

Comment:
can we use `applier` directly? What's their difference?

Reply:
I extract the process of executing physical plan as a function. We can directly call the DataLogApplier.exexcutePhysicalPlan to skip log processing.